lib/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ rest/models/edit_message_args.cpp rest/models/bulk_delete_message_args.cpp
 rest/models/edit_channel_permissions_args.cpp rest/models/create_channel_invite_args.cpp
 rest/models/add_dm_recipient_args.cpp)
 
-include_directories(${CMAKE_SOURCE_DIR}/include)
+include_directories(${CMAKE_SOURCE_DIR}/include ${CMAKE_SOURCE_DIR}/include/disccord)
```

Comment:
I'd rather have them all be consistent because it makes things easier in the future